babel.config.js

```diff
@@ -9,6 +9,7 @@ function resolveAliasPath(relativeToBabelConf) {
 }
 
 const defaultAlias = {
+  '@mui/mui-varnish': resolveAliasPath('./packages/mui-varnish/src'),
```

Comment:
IMO we shouldn't use `@mui`, as we can't publish to their NPM organization.

I think we use `@allenai`, right?

Reply:
due to the way the menus are constructed, in order to have the mui component menu include the varnish components, we need this structure
we might be able to fix int the future